subxt/src/backend/unstable/rpc_methods.rs

```diff
@@ -745,11 +769,16 @@ pub enum TransactionStatus<Hash> {
 
 /// Details of a block that a transaction is seen in.
 #[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+#[cfg_attr(test, derive(Serialize))]
```

Comment:
Should we just expose a `derive(Serialize)` instead of placing it under a cfg(test)? That might turn out to be useful for some developers, wdyt?